addons/l10n_ec/views/account_journal_view.xml

```diff
@@ -7,17 +7,25 @@
             <field name="inherit_id" ref="l10n_latam_invoice_document.view_account_journal_form"/>
             <field name="arch" type="xml">
                 <field name="l10n_latam_use_documents" position="after">
+                	<field name="l10n_ec_require_emission" invisible="1"/>
                     <field name="l10n_ec_entity"
-                           attrs="{'invisible':['|', '|', ('country_code', '!=', 'EC'), ('l10n_latam_use_documents', '=', False), ('type', 'not in', ('sale', 'purchase'))], 'required':[('country_code', '=', 'EC'), ('l10n_latam_use_documents', '=', True), ('type', 'not in', ('sale', 'purchase'))]}"/>
+                           placeholder="001"
+                           attrs="{'invisible':[('l10n_ec_require_emission', '!=', True)],
+                                   'required':[('l10n_ec_require_emission', '=', True)]}"
+                           />
                     <field name="l10n_ec_emission"
-                           attrs="{'invisible':['|', '|', ('country_code', '!=', 'EC'), ('l10n_latam_use_documents', '=', False), ('type', 'not in', ('sale', 'purchase'))], 'required':[('country_code', '=', 'EC'), ('l10n_latam_use_documents', '=', True), ('type', 'not in', ('sale', 'purchase'))]}"
-                    />
+                           placeholder="001"
+                           attrs="{'invisible':[('l10n_ec_require_emission', '!=', True)],
+                                   'required':[('l10n_ec_require_emission', '=', True)]}"
+                           />
                     <field name="l10n_ec_emission_address_id"
-                           attrs="{'invisible':['|', '|', ('country_code', '!=', 'EC'), ('l10n_latam_use_documents', '=', False), ('type', 'not in', ('sale', 'purchase'))], 'required':[('country_code', '=', 'EC'), ('l10n_latam_use_documents', '=', True), ('type', 'not in', ('sale', 'purchase'))]}"
-                    />
-                    <field name="l10n_ec_emission_type"
-                           attrs="{'invisible':['|', '|', ('country_code', '!=', 'EC'), ('l10n_latam_use_documents', '=', False), ('type', 'not in', ('sale', 'purchase'))], 'required':[('country_code', '=', 'EC'), ('l10n_latam_use_documents', '=', True), ('type', 'not in', ('sale', 'purchase'))]}"
-                    />
+                           attrs="{'invisible':[('l10n_ec_require_emission', '!=', True)],
+                                   'required':[('l10n_ec_require_emission', '=', True)]}"
+                           context="{'default_parent_id': company_partner_id, 
+                                     'default_type': 'invoice',
```

Comment:
`default_move_type` ? type is very generic and it will be very complicated to see from where such context key comes from when debugging

Reply:
`default_move_type` is being used in plenty of contexts, is this what we want @jco-odoo @smetl ?

Reply:
@smetl @odoostan The related field is a res.partner, so the type is about the type field on the partner then which is called type and there is not much we can do about it, I guess.

Reply:
I forgot `default_x` is interpreted as a default value for attribute x :facepalm:
Changing it back to `default_type`
```py
type = fields.Selection(
    [('contact', 'Contact'),
        ('invoice', 'Invoice Address'),
        ('delivery', 'Delivery Address'),
        ('private', 'Private Address'),
        ('other', 'Other Address'),
    ], string='Address Type',
    default='contact',
    help="- Contact: Use this to organize the contact details of employees of a given company (e.g. CEO, CFO, ...).\n"
            "- Invoice Address : Preferred address for all invoices. Selected by default when you invoice an order that belongs to this company.\n"
            "- Delivery Address : Preferred address for all deliveries. Selected by default when you deliver an order that belongs to this company.\n"
            "- Private: Private addresses are only visible by authorized users and contain sensitive data (employee home addresses, ...).\n"
            "- Other: Other address for the company (e.g. subsidiary, ...)")
```